src/draive/conversation/realtime/state.py

```
@@ -0,0 +1,54 @@
+
+__all__ = ("RealtimeConversation",)
+
+
+@final
+class RealtimeConversation(State):
+    @classmethod
+    async def start(
+        cls,
+        *,
+        instruction: Instruction | None = None,
+        input: AsyncIterator[ConversationStreamElement],  # noqa: A002
```

Pylint (3.3.7)

[warning] Redefining built-in 'input'

(W0622)